src/Components/Progress.js

```diff
@@ -1,3 +1,15 @@
-export default function Progress() {
-  return null;
+// 
+const Progress = (props) => {
+  return(
+    <section class="progress">
+      <h2>
+       raised <span class="secondary">$335</span> of
+        <span class="secondary">$1000</span>
```

Comment:
you passed the target amount as a prop. How can you use that variable to not have the amount hard coded?